### ISSUE TITLE
Set up static html documentation for R package

### ISSUE THREAD

**Issue by terrytangyuan** [tensorflow/io#26]
We need to scaffold R package documentation website. [pkgdown](https://github.com/r-lib/pkgdown) is a popular choice in R community to generate this automatically through simple configuration. We should at least scaffold the following pages:

- [ ] API reference
- [ ] Tutorials
- [ ] Index page

**Comment by terrytangyuan** [comment]
FYI I'll be working on this soon. Also cc @missaugustina since you are working on docs in https://github.com/tensorflow/io/issues/227 and just realized that you are involved in the R world as well. I don't think there will be conflict with Python docs though since it's totally independent.

**Comment by missaugustina** [comment]
I actually use Rstudio's Tensorflow/Keras libs so yeah I can definitely help here. What's the story behind this Issue? Is the Tensorflow IO repo going to have an R package as an output?

**Comment by terrytangyuan** [comment]
@missaugustina The R package we took for tfio is actually the same as what we did for the other TensorFlow R packages, through [reticulate package](https://github.com/rstudio/reticulate). I believe the code structure would be easier to understand once you get familiar with reticulate. The R package consists of thin wrappers of the Python API. We are definitely lack of documentations and examples though.
